src/components/Slider/Slider.js

```diff
@@ -276,15 +277,15 @@ export default class Slider extends PureComponent {
             <span>{max}</span>
             <span>{maxLabel}</span>
           </span>
-          {!hideTextInput
-            ? <TextInput
-                type={inputType}
-                id="input-for-slider"
-                className="bx-slider-text-input"
-                value={value}
-                onChange={this.handleChange}
-              />
-            : null}
+          {!hideTextInput ? (
```

Comment:
Since the cons case is just `null`, should we just:

```jsx
{!hideTextInput && (
  // ...
)}
```

Reply:
works for me